conans/assets/templates/new_v2_bazel.py

```diff
@@ -28,8 +28,7 @@ def layout(self):
 
     def build(self):
         bazel = Bazel(self)
-        bazel.configure()
-        bazel.build(label="//main:{name}")
+        bazel.build()
```

Comment:
If this new template is not going to work with conan < 1.62, then I think we should add a `conan_min_required` to the template